release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -197,6 +197,15 @@ For more information, see xref:../networking/external_dns_operator/nw-installing
 
 SR-IOV support is now available for xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Pensando DSC cards]. OpenShift SR-IOV is supported, but you must set a static, Virtual Function (VF) media access control (MAC) address using the SR-IOV CNI config file when using SR-IOV.
 
+[id="ocp-4-11-support-for-hosted-control-planes"]
+==== Hosted control planes (Technology preview)
+
+A technology preview is available in {product-title} version 4.11 for hosted control planes. This new deployment model removes the requirement that clusters be self-hosted. Instead, a hosted cluster's control plane exists as pods on another management cluster.
+
+Adapting the OVN networking plug-in to the hosted control planes model requires careful design. OVN provides its own control plane and data store, all of which are external to the hosted cluster.
+
+You must update the OVN deployment model to support split control planes. Likewise, the Network Operator must be able to manage components on both the control plane and the hosted cluster.
```

Comment:
Users don't need to do anything. This is all done for them by our developers. I.e., They get that for free when using hosted control-planes.

I think the release note in that case is informational. For example, with hosted control planes for Red Hat OpenShift, OVN's own control-plane is part hosted away from workloads, and is part of the cluster control plane ... 

Again, please paraphrase as you see fit